doc/gettingstarted.md

```diff
@@ -91,319 +97,555 @@ $ ttn-lw-cli login
 
-Downlinks can be scheduled by adding the `class_bc` flag to the downlink message, which specifies on which gateway(s) the downlink should be scheduled.
+Multicast sessions do not allow uplink. Therefore, you need to explicitly specify the gateway(s) to send messages from, using the `class_b_c` field:
 
 ```json
 {
-    "downlinks": [{
-        "f_port": 15,
-        "frm_payload": "vu8=",
-        "class_b_c": {
-            "gateways": [{
-                "gateway_ids": {
-                    "gateway_id": "gtw1"
-                }
-            }]
+  "downlinks": [{
+    "f_port": 15,
+    "frm_payload": "vu8=",
+    "priority": "NORMAL",
+    "class_b_c": {
+      "gateways": [{
+        "gateway_ids": {
+          "gateway_id": "gtw1"
         }
-    }]
+      }]
+    }
+  }]
 }
 ```
 
-## <a name="webhooks">Using webhooks</a>
+>Note: if you specify multiple gateways, the Network Server will try the gateways in the order specified. The first gateway with no conflicts and no duty-cycle limitation will send the message.
+
+### Listing the downlink queue
 
-The webhooks feature allows the application server to send application related messages to specific HTTP(S) endpoints. Creating a webhook requires you to have an endpoint available as a message sink.
+The stack keeps a queue of downlink messages. Applications can keep pushing downlink messages or replace the queue with a list of downlink messages.
+
+You can see what is in the queue;
 
 ```bash
-$ ttn-lw-cli app webhook set --application-id app1 --webhook-id wh1 --base-url https://example.com/lorahooks --join-accept.path "join" --format "json"
+$ ttn-lw-cli end-devices downlink list app1 dev1
 ```
 
-This will create an webhook `wh1` for the application `app1` with a base URL `https://example.com/lorahooks` and a join-accept path `join`. When a device of the application `app1` joins the network, the application server will do a `POST` request on the endpoint `https://example.com/lorahooks/join` with the following body:
+## <a name="webhooks">Using webhooks</a>
 
-```json
-{
-	"end_device_ids": {
-		"device_id": "dev-simulator",
-		"application_ids": {
-			"application_id": "app1"
-		},
-		"dev_eui": "4200000000000000",
-		"join_eui": "4200000000000000",
-		"dev_addr": "01E9EF6A"
-	},
-	"correlation_ids": ["gs:conn:01D2CSNX7FJVKQPCVG612QF1TX", "gs:uplink:01D2CWCK40JJFVY0J9KXQ2QQYP", "ns:uplink:01D2CWCK41YNDZ16QX3MFY7YAT", "rpc:/ttn.lorawan.v3.GsNs/HandleUplink:01D2CWCK418QERZBCP7AXDGX4J", "as:up:01D2CWCKAB5B148AHB0ED5MCQE"],
-	"join_accept": {
-		"session_key_id": "AWiZxkyDbxhYoP22ceb7SQ=="
-	}
-}
-```
+The webhooks feature allows the Application Server to send application related messages to specific HTTP(S) endpoints. The `json` formatter uses the same format as the MQTT server described above.
```

Comment:
They can get the list of available formatters with `ttn-lw-cli applications webhook get-formats`